COLLABORATOR_GUIDE.md

```diff
@@ -444,6 +444,10 @@ commit logs, ensure that they are properly formatted, and add
 * The commit message text must conform to the
 [commit message guidelines](./CONTRIBUTING.md#step-3-commit).
 
+Run tests (`make test` or `vcbuild test`). Even though there was a successful
```

Comment:
Right now:

`make test` shows up in:

* BUILDING.md
* CONTRIBUTING.md
* doc/guides/building-node-with-ninja.md:

`make -j4 test` shows up in:

* .github/PULL_REQUEST_TEMPLATE.md
* CONTRIBUTING.md
* doc/onboarding-extras.md:

But this doc is the only place where `make -j8 test` shows up.

I think we should standardize on something in a different PR. And in fact, I might go off and do that right...about....now!